cartridges/app_storefront_pdp_extend/cartridge/templates/default/product/productDetails.isml

```diff
@@ -0,0 +1,160 @@
+<isdecorate template="common/layout/page">
+
+    <isscript>
+        var ProductMgr = require("dw/catalog/ProductMgr");
```

Comment:
Is this used?

Reply:
Yes, i use it in addToCartProduct.isml to check whether the current product is inStock or outOfStock, so i can display the proper element (addToCartBtn or subscriptionForm)

Reply:
<div id="addToCartButton" class="col-sm-12 ${!ProductMgr.getProduct(product.id).availabilityModel.inStock ? "d-none" : ""} ">

Reply:
` <div id="addToCartButton" class="col-sm-12 ${!ProductMgr.getProduct(product.id).availabilityModel.inStock ? "d-none" : ""} ">`

Reply:
As Krasi mentioned, restrain from using it in the template